tests/test_archivesspace_client.py

```diff
@@ -393,3 +402,14 @@ def test_contentless_notes():
     collections = client.find_collections()
     assert collections[-1]['notes'][0]['type'] == 'bioghist'
     assert collections[-1]['notes'][0]['content'] == ''
+
+
+def test_escaping_solr_queries():
+    client = ArchivesSpaceClient(**AUTH)
+    query = '"quotes"'
+    # Test escaping single characters
+    assert client._escape_solr_query(query, field='identifier') == '\\"quotes\\"'
+    assert client._escape_solr_query(query, field='title') == '\\\\"quotes\\\\"'
+    # And double characters, which require only one set of escape tokens
+    assert client._escape_solr_query('&&test', field='identifier') == '\\&&test'
```

Comment:
This should be the same as `r'\&&test'`, which might be easier to read (only one layer of escaping).